job-descriptions/software-engineer-security.md

```diff
@@ -38,15 +38,14 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
 ## Interview process
 
 1. You [apply here](https://jobs.lever.co/sourcegraph/c36db3e1-0ece-465d-ad7c-1eb6de9a4b22/apply).
-1. You speak with the VP of Engineering for 30 minutes. To prepare:
+1. You speak with the Security manager for 30 minutes. To prepare:
    - Read through [our handbook](https://github.com/sourcegraph/about) to learn more about how we operate and to find answers to common questions that you might have.
-   - Review the RFC template in the next step and write down any questions that you have.
-1. You write your first Sourcegraph [RFC](https://about.sourcegraph.com/handbook/communication/rfcs) by creating a new Google Doc using [this template](https://docs.google.com/document/d/1ol7aVXuXB7XL4DorOoxoDsaSyFI9Pv4Bcc1zfo-iLtw/edit#). We will review your RFC and may ask questions via comments directly on the document.
-1. You speak with the Sourcegraph Cloud engineering manager for 1 hour about your RFC.
-1. We schedule 4 hours of remote interviews over video chat across multiple days.
+1. You write a short security assessment, taking less than **30m** of your time, explaining a security vulnerability.
```

Comment:
Can you flesh this out more? Is this a security assessment about Sourcegraph or could it be about a past security threat? Can a candidate share something they have already written?

Reply:
I'm going to dummy up a template, much like the RFC.  The idea is that a candidate would explain a vulnerability for which we provide the appropriate research.  The candidate will help explain the vulnerability **in plain English**.  Given the information we provide, they would ideally explain the impact of the vulnerability to the Sourcegraph product.  I should have it ready tomorrow afternoon.